guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -166,21 +109,68 @@ Finally, you can calculate your estimated generated revenue share by multiplying
 estimated revenue share you generate = $22,500 x 0.025% = $5.625
 ```
 
+### Risks Associated with Holding JLP
 
+1. **Bull Markets**
 
-### Virtual Price, Market Price and AUM Limit
+During a bull market, JLP may not outperform SOL, ETH, or BTC. This is because the pool is made of a mix of stable and non-stable tokens. While this factor is less of a risk, it is worth noting that during a bull market, the JLP token will move slower than some of the coins it contains. 
```

Comment:
```suggestion
During a bull market, JLP may not outperform SOL, ETH, or BTC. This is because the pool is made of a mix of stable and non-stable tokens.
```

We can probably remove this since it's roughly the same as first two lines, WDYT?